.github/workflows/ci-suite.yml

```diff
@@ -186,25 +186,25 @@ jobs:
     steps:
     - name: Set General__UseBasicWatchdog
       if: ${{ matrix.watchdog-type == 'Basic' }}
-      run: echo "::set-env name=General__UseBasicWatchdog::true"
+      run: echo "General__UseBasicWatchdog=true" >> $GITHUB_ENV
 
     - name: Set TGS4_TEST_CONNECTION_STRING
-      run: echo "::set-env name=TGS4_TEST_CONNECTION_STRING::Server=(localdb)\MSSQLLocalDB;Integrated Security=true;Initial Catalog=TGS_${{ matrix.watchdog-type }}_${{ matrix.configuration }};Application Name=tgstation-server"
+      run: echo "TGS4_TEST_CONNECTION_STRING=Server=(localdb)\MSSQLLocalDB;Integrated Security=true;Initial Catalog=TGS_${{ matrix.watchdog-type }}_${{ matrix.configuration }};Application Name=tgstation-server" >> $GITHUB_ENV
```

Comment:
This change is sus methinks.

Reply:
grr, why wasn't this using Write-Host then